charts/camunda-platform/README.md

```diff
@@ -244,6 +244,7 @@ Check out the default [values.yaml](values.yaml) file, which contains the same c
 | | `identity.keycloak.auth.adminUser` |    Can be used to configure admin user to access existing Keycloak | `""` |
 | | `identity.keycloak.auth.existingSecret` |    Can be used to configure existing Secret object which has admin password to access existing Keycloak | `""` |
 | | `identity.keycloak.auth.existingSecretKey` |    Can be used to configure the key inside existing Secret object which has admin password to access existing Keycloak | `"admin-password"` |
+| | `identity.keycloak.postgresql.primary.sidecars` | Can be set to deploy extra containers in the same postgresql pod | `[]` |
```

Comment:
No need to add it for any 3rd party deps.

Reply:
Yeah I think we can expect people to look at the dependency keycloak helm chart if they want to configure that. I'll remove this and the relevant values.yaml section.

Reply:
I removed references to attaching sidecars to keycloak and postgresql.